projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1303,13 +1265,9 @@ public function should_submit_post_pre_checks( $post ) {
 		// - API/XML-RPC Test Posts
 		if (
 			(
-				defined( 'XMLRPC_REQUEST' )
-			&&
-				XMLRPC_REQUEST
+			( defined( 'XMLRPC_REQUEST' ) && XMLRPC_REQUEST )
 			||
-				defined( 'APP_REQUEST' )
-			&&
-				APP_REQUEST
+			( defined( 'APP_REQUEST' ) && constant( 'APP_REQUEST' ) )
```

Comment:
This is just a formatting change to fix PHPCS errors that bother me with red warnings all the time.